### PR TITLE
Enable static test runner

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR enables static test runner which can validate `sample_event.json` files.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by mtojek** [comment]
jenkins run the tests please

File: packages/aws/manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: aws
 title: AWS
-version: 0.4.0
+version: 0.4.1
```
Comment (ycombinator) [inline packages/aws/manifest.yml:4]: Do we need to bump up the version?

**Review by ycombinator — COMMENTED** [review]
Changes in packages LGTM, except I'm wondering if we want to bump up the `version`s in the package's `manifest.yml` files for these changes. They could go out with other changes in the future, no?

**Comment by mtojek** [comment]
> Changes in packages LGTM, except I'm wondering if we want to bump up the versions in the package's manifest.yml files for these changes. They could go out with other changes in the future, no?

I would say that this PR cleans up the package content which is shown to the customer (sample events) and we don't have any publishing timeline for these packages. If you don't mind, I would be for releasing them due to consistency reasons (all sample events are looking the same).

**Comment by ycombinator** [comment]
> I would be for releasing them due to consistency reasons (all sample events are looking the same).

Ah, okay, that makes sense then. ++ to upversioning.

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**